package.json

```diff
@@ -12,7 +12,10 @@
     "mcritic": "mcritic packages/ --ignore '**/{browserslist-config-clay-components,claycss.com,generator-metal-clay,node_modules}/**'",
     "prettier": "prettier-eslint packages/clay-*/src/*.js packages/clay-*/src/**/*.js",
     "start": "http-server . -p 4000",
-    "test": "npm run build && npm run jest && npm run a11y"
+    "test": "npm run build && npm run jest && npm run a11y",
+    "electric": "cd packages/clay && npm run build && cd ../clay-charts && npm run compile && npm run sass",
```

Comment:
Hey @julien, even if we might incur into some overhead now, I assume at some point we'll want to add other components as well, so why not just have these do:

```javascript
...
"electric": "npm run build && cd packages/claycss.com && el",
"web": "npm run electric -- run"
"deploy": "npm run electric -- deploy"
...
```

Reply:
Yes, good point.